packages/regal/lib/src/widgets/regal_date_picker.dart

```diff
@@ -121,9 +121,6 @@ class _CustomDatePickerState extends State<CustomDatePicker> {
                               if (widget.onSelected != null) {
                                 final dateTime = _tempSelected ?? _selected;
 
-                                if (dateTime == null) {
```

Comment:
removed?

Reply:
`_selected` is the null fallback, which is non-nullable.